integrationTests/cypress/integration/redmine_administration.feature

```diff
@@ -0,0 +1,69 @@
+Feature: Administrative Procedures
+
+  @requires_testuser
+  Scenario: cas user + no redmine user => login => create normal redmine account
+    Given the user is not member of the admin user group
+    And the user has no internal redmine account
+    And the user is logged out of the CES
+    When the user logs into the CES
+    Then the user has an internal redmine account with default privileges
+
+  @requires_testuser
+  Scenario: cas user (admin) + no redmine user => login => create admin redmine account
+    Given the user is member of the admin user group
+    And the user has no internal redmine account
+    And the user is logged out of the CES
+    When the user logs into the CES
+    Then the user has an internal redmine account with admin privileges
+
+  @requires_testuser
+  Scenario: cas user + internal default redmine user => login => preserves default account
```

Comment:
Suggestion:
"cas user + internal default redmine user => login => preserves default account"
"A normal CAS user, with existing normal redmine account logs in to redmine where no changes to his permissions takes place."